pkg/store/store.go

```diff
@@ -191,7 +191,7 @@ func mutate(ctx context.Context, s storeLoader, builder qb.Builder, values ...in
 	return nil
 }
 
-func (ds delegatingStore) Check(ctx context.Context, table *typedef.Table, builder qb.Builder, values ...interface{}) error {
+func (ds delegatingStore) Check(ctx context.Context, table *typedef.Table, builder qb.Builder, lastAttempt bool, values ...interface{}) error {
```

Comment:
this variable name should reflect what it is actually do, example: `detailedDiff`, `returnDetails` or opposite of that `simpleDiff`, `returnSimple`

Reply:
Done